src/utils/explorer-api.js

```diff
@@ -39,8 +38,31 @@ export async function getTransactions(accountId = '') {
             }
         }
     ));
-    return tx.map((t) => ({
-        ...t,
-        actions: JSON.parse(t.actions)
-    }))
+    
+    tx.map(async t => {
+        const actions = await new Promise((require, reject) => wamp.call(
+            'com.nearprotocol.testnet.explorer.select',
+            [
+                `SELECT actions.action_type as kind, 
+                        actions.action_args as args
+                FROM actions
+                WHERE actions.transaction_hash = :hash
+                ORDER BY actions.action_index`,
+                  {
+                    hash: t.hash
+                  }
+            ]
+        ))
```

Comment:
This doesn’t call resolve/reject so cannot work properly. I suggest wrapping wamp.call into helper function which handles promise creation automatically.